synchronization/src/main/java/de/cyface/synchronization/SyncPerformer.java

```diff
@@ -0,0 +1,125 @@
+package de.cyface.synchronization;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.ProtocolException;
+import java.net.URL;
+
+/**
+ * Performs the actual synchronisation with a provided server, by uploading meta data and a file containing
+ * measurements.
+ * 
+ * @author Klemens Muthmann
+ * @version 1.0.0
+ * @since 2.0.0
```

Comment:
Version < Since? Does this make sense? If not, you might wanna check if this typo occurs in this PR multiple times ...

Reply:
The version of the Framework is 2.0.0. (Our old app is considered 1.0.0). This call was created in version 2.0.0 of the framework so @since is 2.0.0. The version of the class is still 1.0.0 since it was created here. @since and @version have nothing to do with each other.